metron-platform/metron-indexing/src/main/config/zookeeper/indexing/error.json

```diff
@@ -0,0 +1,17 @@
+{
+  "hdfs" : {
```

Comment:
Yes that is correct.  There is an "error_type" type fields that distinguishes between the different type of errors.  If you think there is a strong reason to make "source.type" the "error_type" field and add indexing configs for all the different types we can do that.